tidalapi/bin/generate-api-files.py

```diff
@@ -174,6 +174,14 @@ def main():
         logging.error("Error: Failed to generate files.")
         sys.exit(result.returncode)
 
+    result = subprocess.run(
+        [f"{sdk_root}/static-analysis/run-ktlint.sh", "-F", "-g", "-d", f"{project_root}/src"])
+
+    if result.returncode != 0:
+        logging.error("ktlint failed, but continuing with the rest of the script.")
+
+    target_line = "import com.tidal.sdk.tidalapi.generated.infrastructure.CollectionFormats.*"
```

Comment:
Note that `ktlint` cannot fix these imports (* imports of types provided by us), so we manually remove them anywhere they show up as we don't need them, but `openapi-generator` just puts it everywhere